pkg/network/network.go

```diff
@@ -57,6 +58,8 @@ const (
 
 	// IstioOutboundIPRangesKey is the name of the configuration entry
 	// that specifies Istio outbound ip ranges.
+	//
+	// OBSOLETE: This will be completely removed once v1.4 is released.
```

Comment:
1.4? Do you mean 0.14?

Reply:
Yes, and I realized that I should not put the version number but just say "future release". Updated.